tasks/mpi/korovin_n_matrix_multiple_cannon/func_tests/main.cpp

```diff
@@ -0,0 +1,496 @@
+// Copyright 2023 Nesterov Alexander
+#include <gtest/gtest.h>
+
+#include <boost/mpi/communicator.hpp>
+#include <boost/mpi/environment.hpp>
+#include <vector>
+
+#include "mpi/korovin_n_matrix_multiple_cannon/include/ops_mpi.hpp"
+
+TEST(korovin_n_matrix_multiple_cannon_mpi, matrix_1x1) {
+  boost::mpi::communicator world;
+  int numRowsA = 1;
+  int numColsA_RowsB = 1;
+  int numColsB = 1;
+  std::vector<double> A = {7};
+  std::vector<double> B = {3};
+  std::vector<double> expected_C = {21};
+  std::vector<double> C_mpi(numRowsA * numColsB, 0.0);
+  std::vector<double> C_seq(numRowsA * numColsB, 0.0);
+
+  std::shared_ptr<ppc::core::TaskData> taskData = std::make_shared<ppc::core::TaskData>();
+  if (world.rank() == 0) {
+    taskData->inputs_count.emplace_back(numRowsA);
+    taskData->inputs_count.emplace_back(numColsA_RowsB);
+    taskData->inputs_count.emplace_back(numColsB);
+    taskData->inputs.emplace_back(reinterpret_cast<uint8_t*>(A.data()));
+    taskData->inputs.emplace_back(reinterpret_cast<uint8_t*>(B.data()));
+    taskData->outputs.emplace_back(reinterpret_cast<uint8_t*>(C_mpi.data()));
+  }
+
+  korovin_n_matrix_multiple_cannon_mpi::TestMPITaskParallel testTask(taskData);
+  ASSERT_EQ(testTask.validation(), true);
```

Comment:
Please use ASSERT_TRUE
```suggestion
  ASSERT_TRUE(testTask.validation());
```

Reply:
Thanks, done